darwin/cli_functions.py

```diff
@@ -652,6 +652,20 @@ def set_file_status(dataset_slug: str, status: str, files: List[str]) -> None:
         _error(f"No dataset with name '{e.name}'")
 
 
+def delete_files(dataset_slug: str, files: List[str]) -> None:
+    client: Client = _load_client(dataset_identifier=dataset_slug)
+    try:
+        dataset: RemoteDataset = client.get_remote_dataset(dataset_identifier=dataset_slug)
+        items: Iterator[DatasetItem] = dataset.fetch_remote_files({"filenames": ",".join(files)})
+        if not secure_continue_request():
+            print("Cancelled.")
```

Comment:
I personally think we're fine without this `print` call

Reply:
The print call is standard in other places where we ask for confirmation as well, that's why I kept it.
I will look into the `-y` flag.

Reply:
Resolved over Slack call.